app/scripts/modules/kubernetes/src/serverGroup/serverGroupTransformer.service.spec.ts

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+import { KubernetesV2ServerGroupTransformer } from './serverGroupTransformer.service';
+import { Application } from '@spinnaker/core';
+import { IKubernetesServerGroup } from '../interfaces';
+
+describe('KubernetesV2ServerGroupTransformer', function() {
+  it('normalized server group name', async () => {
+    const transformer = new KubernetesV2ServerGroupTransformer();
+    const ACCOUNT = 'myaccount';
+    const LOCATION = 'myns';
+    const SG_MANAGER = 'sgmanager';
+    const KIND = 'daemonSet';
+
+    const sgMgrRef = { account: ACCOUNT, location: LOCATION, name: SG_MANAGER };
+    const sgAccount: IKubernetesServerGroup = ({
+      account: ACCOUNT,
+      name: 'myservergroup',
+      region: LOCATION,
+      serverGroupManagers: [sgMgrRef],
+    } as unknown) as IKubernetesServerGroup;
+
+    const data = [{ name: `${KIND} ${SG_MANAGER}`, region: LOCATION, account: ACCOUNT, kind: KIND }];
+    const app = new Application('test', null, []);
+    app.getDataSource = () => {
+      return {
+        ready: () => Promise.resolve(data),
+        data,
+      } as any;
+    };
+
+    const sg = await transformer.normalizeServerGroup(sgAccount, app);
+    expect(sg).not.toBe(null);
+    expect(sg.serverGroupManagers.length).toBe(1);
+    expect(sgMgrRef.name).toBe('daemonSet sgmanager');
+  });
+
+  it('normalized server group different account', async () => {
```

Comment:
it's semi-conventional to try to write these such that the test description makes a sentence with `it` e.g. `it('normalizes the server group name')`

Reply:
The `d` is too close to the `s` :) Thanks!